developer-workflow/grammar.rst

```diff
@@ -42,7 +42,8 @@ Note: sometimes things mysteriously don't work.  Before giving up, try ``make cl
   Then run ``make regen-ast`` to regenerate
   :cpy-file:`Include/internal/pycore_ast.h` and :cpy-file:`Python/Python-ast.c`.
 
-* :cpy-file:`Parser/tokenizer.c` contains the tokenization code.
+* :cpy-file:`Parser/lexer/` and :cpy-file:`Parser/tokenizer/` contain the
+  tokenization code.
```

Comment:
```suggestion
* :cpy-file:`Parser/lexer/` contain the tokenization code.
```

Reply:
Done. Thanks @lysnikolaou and @AlexWaygood for the review and suggestions :)